src/Consolonia.Themes/Templates/Controls/ListBoxItem.axaml

```diff
@@ -10,10 +10,11 @@
                 Value="{DynamicResource ThemeTransparentBrush}" />
         <Setter Property="Template">
             <ControlTemplate>
-                <Panel Background="{TemplateBinding Background}">
+                <Panel Background="Transparent">
```

Comment:
Do we know why ThemeTransparentBrush did not do the job?

Reply:
no.

Reply:
huh. I think the scrollviewer fix makes the listbox work without any changes.

Reply:
I don't know why that theme brush didn't work. When I use hard-coded "Transparent" it works.